package.json

```diff
@@ -55,6 +55,7 @@
     "cheerio": "^1.0.0-rc.2",
     "codecov": "^3.2.0",
     "copyfiles": "^2.1.0",
+    "cucumber": "^5.1.0",
```

Comment:
Shouldn't this be a dependency instead of a devDependency?

Reply:
it doesn't belong there anyway .. removed it.